tests/test_interact.py

```diff
@@ -41,7 +41,9 @@ def setUp(self):
         else:
             env['PYTHONPATH'] = self.project_dir
 
-        self.interact_py = ('{sys.executable} interact.py'.format(sys=sys))
+        self.interact_py = ('{self.PYTHONBIN} interact.py'.format(self=self))
+        if self.PYTHONBIN != sys.executable:
+            self.interact_py += ' --pythonbin {self.PYTHONBIN}'.format(self=self)
```

Comment:
I'm unsure why this bit is needed - surely if you run the subprocess with `self.PYTHONBIN`, then its `sys.executable` will always be the Python you want?

Reply:
That's possible, actually, I'll test without this part of this change.

Reply:
Removed this and the related unnecessary change in interact.py from the pull request.